tests/php/test-includes-admin-functions.php

```diff
@@ -125,7 +126,7 @@ public function get_data_for_test_amp_get_customizer_url() {
 					Option::THEME_SUPPORT => AMP_Theme_Support::READER_MODE_SLUG,
 					Option::READER_THEME  => ReaderThemes::DEFAULT_READER_THEME,
 				],
-				function () {
+				'assert' => function () {
```

Comment:
Was this causing an error before or is it just a bug? It's strange that there's an associative array mixed with a numerical index.

Reply:
In PHP 8 it maps to a named parameter.